vmdk_plugin/vmdkops/vmdkops.go

```diff
@@ -105,7 +105,7 @@ func (v VmdkOps) Get(name string) (map[string]interface{}, error) {
 
 	err = json.Unmarshal(str, &statusMap)
 	if err != nil {
-		return nil, err
+		log.Debugf("vmdkOps.Get failed decoding volume status for name=%s", name)
```

Comment:
It should be warning , not debug. Admin cant do much about it but at least needs to know about it since it is why default would be  applied.